doc/nrf/app_dev/ap_protect/index.rst

```diff
@@ -118,19 +118,21 @@ Based on the available implementation types, you can configure the access port p
 
    * - AP-Protect state
      - Related Kconfig option in the |NCS|
-     - Description
+     - Description of the AP-Protect state
      - AP-Protect implementation type
    * - Locked
      - :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` for Secure AP-Protect)
-     - In this state, CPU writes to enable and lock AP-Protect. UICR is not modified.
+     - In this state, CPU uses the MDK system start-up file to write to enable and lock AP-Protect. UICR is not modified.
      - Hardware and software
    * - Authenticated
      - :kconfig:option:`CONFIG_NRF_APPROTECT_USER_HANDLING` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` for Secure AP-Protect)
-     - In this state, AP-Protect is left enabled and it is up to the user handler to unlock the device if needed.
+     - In this state, AP-Protect is left enabled and it is up to the user-space code to handle unlocking the device if needed.
+       The MDK will close the application subsystem access port AHB-AP, but not lock it, so the AHB-AP can be reopened by the firmware.
```

Comment:
```suggestion
       The MDK will close the debug AHB-AP, but not lock it, so the AHB-AP can be reopened by the firmware.
```
This is per domain, so shouldn't mention application subsystem here (network will operate independently from application core for example)

May be that AHB-AP is not sufficiently described at this point, maybe we want to add a description further up if we have a concepts section. Alternatively we can drop AHB-AP and just talk about debugging being closed or something.